### ISSUE TITLE
Nothing on Accessibility Options

### ISSUE THREAD

**Issue by MarjaE2** [MicrosoftDocs/WSL#2354]
### Documentation Issue

"No results. Sorry, we couldn't find anything that fit your search in WSL. You can still view all results in Microsoft Learn."

### Link to documentation page

https://learn.microsoft.com/en-us/search/?scope=WSL&terms=accessibility

### Suggested Improvements

Linux has several front-ends for its settings, accessibility and otherwise. I haven't tried all of them, but gnome-control-center will not run in WSL2; cinnamon-settings will run, but it won't all work. The Themes panel allows users to change scrollbar settings, and these changes work. The Effects panel opens, but if users turn off animation effects here, the changes don't work; the animations continue, everywhere. The Keyboard panel crashes.

If WSL2 has its own tool for these settings, the documentation should discuss this.

If WSL2 relies on existing front-ends, the doc should discuss which ones are compatible with it.

I think accessibility settings should be an especially high priority, but most of these settings are accessibility settings for some users.

For example, effects settings, cursor blink rate settings, and overlay scrollbar settings can be important for users who are sensitive to animation and flashing.

**Comment by craigloewen-msft** [comment]
WSL does not have its own accessibility settings. These are done through other accessibility settings on Windows. For example when you interact with `wsl.exe` you use Windows Terminal, and when you use GUI apps you can interact with Linux based screen reader tools etc. So we do not call out anything immediate for accessibility in the WSL docs themselves as the accessibility content is all actually rooted in other products.

**Comment by MarjaE2** [comment]
> These are done through other accessibility settings on Windows.

They aren't, though.

Windows settings to block animation, blinking cursor animation, hide-the-scrollbar animation, etc. do not block this animation in wsl apps, making them unusable without the Linux settings.
